docs/development/submitting-changes.rst

```diff
@@ -0,0 +1,84 @@
+Submitting Changes
+==================
+
+This page details how to make and submit changes to the `j5` codebase.
+
+Repositories
+------------
+
+The main repository for `j5` is available on GitHub_, and Pull Requests should be submitted there.
+
+There is an additional repository available on GitLab_, although this is a mirror of the GitHub repo, and changes
+should not be submitted there.
+
+.. _GitHub: https://github.com/j5api/j5
+.. _GitLab: https://gitlab.com/j5api/j5
+
+Publishing Changes
+------------------
+
+The first step to submit changes is to publish them so that other contributors can review and give feedback.
+
+The mechanism to publish your changes depends on whether you have write access to the repository. Write access
+is granted by the Steering Council to recurrent, or trusted contributors.
+
+* Contributors with write access should publish a branch on the repository.
+* Contributors without write access should publish as a branch on a fork of the repository.
+
+Pull Requests
+-------------
+
+The next stage is to submit a pull request (PR) to the GitHub repository. The PR will be used to give feedback on,
+and discuss the changes with the contributor.
+
+When making your PR, consider the following:
+
+* Why do you want to make these changes?
+* Which parts of the codebase do your changes affect?
+* Have you updated the relevant documentation?
+* Are your changes breaking to the external API?
```

Comment:
Could we do something similar to the [node repo](https://github.com/nodejs/node/labels?utf8=%E2%9C%93&q=semver) and add  `semver-patch`, `semver-minor`, and `semver-major` labels?

Reply:
I like this idea.

Reply:
e94ca71